python/in_toto_attestation/v1/resource_descriptor_pb2.py

```diff
@@ -14,201 +14,22 @@
-_RESOURCEDESCRIPTOR_ANNOTATIONSENTRY.containing_type = _RESOURCEDESCRIPTOR
-_RESOURCEDESCRIPTOR.fields_by_name['digest'].message_type = _RESOURCEDESCRIPTOR_DIGESTENTRY
-_RESOURCEDESCRIPTOR.fields_by_name['annotations'].message_type = _RESOURCEDESCRIPTOR_ANNOTATIONSENTRY
-DESCRIPTOR.message_types_by_name['ResourceDescriptor'] = _RESOURCEDESCRIPTOR
-_sym_db.RegisterFileDescriptor(DESCRIPTOR)
-
-ResourceDescriptor = _reflection.GeneratedProtocolMessageType('ResourceDescriptor', (_message.Message,), {
-
-  'DigestEntry' : _reflection.GeneratedProtocolMessageType('DigestEntry', (_message.Message,), {
-    'DESCRIPTOR' : _RESOURCEDESCRIPTOR_DIGESTENTRY,
-    '__module__' : 'in_toto_attestation.v1.resource_descriptor_pb2'
-    # @@protoc_insertion_point(class_scope:in_toto_attestation.v1.ResourceDescriptor.DigestEntry)
-    })
-  ,
-
-  'AnnotationsEntry' : _reflection.GeneratedProtocolMessageType('AnnotationsEntry', (_message.Message,), {
-    'DESCRIPTOR' : _RESOURCEDESCRIPTOR_ANNOTATIONSENTRY,
-    '__module__' : 'in_toto_attestation.v1.resource_descriptor_pb2'
-    # @@protoc_insertion_point(class_scope:in_toto_attestation.v1.ResourceDescriptor.AnnotationsEntry)
-    })
-  ,
-  'DESCRIPTOR' : _RESOURCEDESCRIPTOR,
-  '__module__' : 'in_toto_attestation.v1.resource_descriptor_pb2'
-  # @@protoc_insertion_point(class_scope:in_toto_attestation.v1.ResourceDescriptor)
-  })
-_sym_db.RegisterMessage(ResourceDescriptor)
-_sym_db.RegisterMessage(ResourceDescriptor.DigestEntry)
-_sym_db.RegisterMessage(ResourceDescriptor.AnnotationsEntry)
-
-
-DESCRIPTOR._options = None
-_RESOURCEDESCRIPTOR_DIGESTENTRY._options = None
-_RESOURCEDESCRIPTOR_ANNOTATIONSENTRY._options = None
+DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n0in_toto_attestation/v1/resource_descriptor.proto\x12\x16in_toto_attestation.v1\x1a\x1cgoogle/protobuf/struct.proto\"\x83\x03\n\x12ResourceDescriptor\x12\x0c\n\x04name\x18\x01 \x01(\t\x12\x0b\n\x03uri\x18\x02 \x01(\t\x12\x46\n\x06\x64igest\x18\x03 \x03(\x0b\x32\x36.in_toto_attestation.v1.ResourceDescriptor.DigestEntry\x12\x0f\n\x07\x63ontent\x18\x04 \x01(\x0c\x12\x18\n\x10\x64ownloadLocation\x18\x05 \x01(\t\x12\x11\n\tmediaType\x18\x06 \x01(\t\x12P\n\x0b\x61nnotations\x18\x07 \x03(\x0b\x32;.in_toto_attestation.v1.ResourceDescriptor.AnnotationsEntry\x1a-\n\x0b\x44igestEntry\x12\x0b\n\x03key\x18\x01 \x01(\t\x12\r\n\x05value\x18\x02 \x01(\t:\x02\x38\x01\x1aK\n\x10\x41nnotationsEntry\x12\x0b\n\x03key\x18\x01 \x01(\t\x12&\n\x05value\x18\x02 \x01(\x0b\x32\x17.google.protobuf.Struct:\x02\x38\x01\x42&Z$github.com/in-toto/attestation/go/v1b\x06proto3')
+
+_builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, globals())
+_builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, 'in_toto_attestation.v1.resource_descriptor_pb2', globals())
+if _descriptor._USE_C_DESCRIPTORS == False:
+
+  DESCRIPTOR._options = None
+  DESCRIPTOR._serialized_options = b'Z$github.com/in-toto/attestation/go/v1'
```

Comment:
Still trying to figure out why `protoc` places this here in all python bindings.

Reply:
I think it's fine. It's essentially because of https://github.com/in-toto/attestation/blob/main/protos/in_toto_attestation/v1/statement.proto#L8. I think they pull in everything and it's ignored in Python.

Reply:
Ok! I wasn't able to find out anything about this searching online.